mteb/models/nomic_models.py

```diff
@@ -87,6 +87,17 @@ def encode(  # type: ignore
     open_weights=True,
     revision="b0753ae76394dd36bcfb912a46018088bca48be0",
     release_date="2024-02-10",  # first commit
+    n_parameters=137_000_000,
+    memory_usage=None,
+    max_tokens=8192,
+    embed_dim=768,
+    license="apache-2.0",
+    reference="https://huggingface.co/nomic-ai/nomic-embed-text-v1.5",
+    similarity_fn_name="cosine",
+    framework=["Sentence Transformers", "PyTorch"],
+    use_instructions=True,
```

Comment:
Does only specifying the task count as instruction?

Reply:
yes